provisioning/node.yml

```diff
@@ -1,7 +1,18 @@
 ---
-- hosts: all
+- hosts: node-*
   become: yes
   tasks:
     - name: Placeholder task
       debug:
         msg: "Provisioning for {{ inventory_hostname }} is not yet implemented."
+    - name: Generate Kubernetes join command on controller
+      command: kubeadm token create --print-join-command
+      register: join_command
+      delegate_to: ctrl
```

Comment:
I would suggest using ansible vars like we're doing in ctrl.yml
`"{{ control_node_name }}"`